### PR TITLE
docs(toh-6): Upgraded http tutorial to new router

### PR DESCRIPTION
**Pull request by brandonroberts**
Builds on top of #1735, which upgrades the routing tutorial

### CODE REVIEW

**Comment by NejcZdovc** [comment]
@brandonroberts Shouldn't there be no slashes at the beginning of the router link?

**Comment by Foxandxss** [comment]
There is a broken test. Please check :)

**Comment by brandonroberts** [comment]
@NejcZdovc no. No leading slashes is only for the route configuration. Leading slashes in the routerLink or navigation means the routerLink is absolute. If you don't lead with a slash, it means you are navigating relative to where you are.

**Comment by NejcZdovc** [comment]
@brandonroberts Sorry, you are right I was too fast :)

**Comment by Foxandxss** [comment]
This needs rebase.

**Comment by brandonroberts** [comment]
Rebased

File: public/docs/ts/latest/tutorial/toh-pt6.jade [hunk 1]
```diff
@@ -360,6 +360,7 @@ block review
 
 .l-main-section
 :marked
+<<<<<<< HEAD
```
Comment (Foxandxss) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: This ended here.

File: public/docs/ts/latest/tutorial/toh-pt6.jade [hunk 2]
```diff
@@ -395,6 +396,47 @@ block filetree
       .file systemjs.config.json
       .file tsconfig.json
       .file typings.json
+=======
```
Comment (Foxandxss) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: ditto

File: public/docs/ts/latest/tutorial/toh-pt6.jade [hunk 3]
```diff
@@ -395,6 +396,47 @@ block filetree
...
+  .file node_modules ...
+  .file typings ...
+  .file index.html
+  .file package.json
+  .file styles.css
+  .file sample.css
+  .file systemjs.config.json
+  .file tsconfig.json
+  .file typings.json
+>>>>>>> docs(toh-6/ts): Upgraded http tutorial to use new router
```
Comment (Foxandxss) [inline public/docs/ts/latest/tutorial/toh-pt6.jade]: this too.

**Comment by brandonroberts** [comment]
Issues fixed

**Comment by naomiblack** [comment]
merged!

**Pull request closed without merging**